lib/active_yaml/base.rb

```diff
@@ -18,9 +18,15 @@ def extension
       end
 
       private
+if RUBY_VERSION > "3.1.0"
```

Comment:
@kbrock 
I guess this change is to support Psych v4, but Psych v4 is also available in <Ruby 3.1. So, I think we should use `Psych::VERSION` instead of `RUBY_VERSION`.